### DISCUSSION TITLE
[Question] How to get first and last data using group by

### DISCUSSION THREAD

**Discussion by tarikhagustia** [opensource9ja/danfojs#663] [General]
hi i'm looking for this solution [https://stackoverflow.com/questions/62373692/how-do-you-convert-1-minute-open-high-low-close-data-to-another-timeframefx-5](https://stackoverflow.com/questions/62373692/how-do-you-convert-1-minute-open-high-low-close-data-to-another-timeframefx-5) using this library. but i can't find first and last data grouping.

**Comment by steveoni** [comment]
@tarikhagustia 

for any groupby variable you should be able to do this

```js
let grpby = df.groupby("column")
grpby.first() // first of the grouping
grpby.last() // last of the grouping
```

**Comment by tarikhagustia** [comment]
> @tarikhagustia
> 
> for any groupby variable you should be able to do this
> 
> ```js
> let grpby = df.groupby("column")
> grpby.first() // first of the grouping
> grpby.last() // last of the grouping
> ```

but how to use in `.agg` function?
